docs/read_the_docs/source/workflow_inputs/create_characteristics_rst.rb

```diff
@@ -46,4 +112,46 @@ def to_underscore_case
   write_subsection(f, row, 'Created by', '*')
   write_subsection(f, row, 'Source', '*')
   write_subsection(f, row, 'Assumption', '*')
```

Comment:
Assumption**s**? Handle this on the resstock-estimation side. These should probably not be hard-coded here.